blazor/3D-chart/accessibility.md

```diff
@@ -0,0 +1,93 @@
+
+WAI-ARIA(Accessibility Initiative - Accessible Rich Internet Applications) defines a way to increase the accessibility of web pages, dynamic content, and user interface components developed with AJAX, HTML, JavaScript, and related technologies. ARIA provides additional semantics to describe the role, state, and functionality of web components.
+
+Element |Default description
+-----|-----
+Datalabel |Reads the Point y value.
+Legend |Click to show or hide the series.
+Axis Title |Reads the axis title.
+Chart Title |Reads the chart title.
+Series Points |Reads the Point x: Point y value.
+
+The Blazor 3D Chart component followed the [WAI-ARIA](https://www.w3.org/WAI/ARIA/apg/patterns/alert/) patterns to meet the accessibility. The following ARIA attributes are used in the Blazor 3D Chart component:
+
+* img (role)
+* button (role)
+* region (role)
+* aria-label (attribute)
+* aria-hidden (attribute)
+* aria-pressed (attribute)
+
+## Keyboard navigation
+
+The Blazor 3D Chart component followed the [keyboard interaction](https://www.w3.org/WAI/ARIA/apg/patterns/alert/#keyboardinteraction) guideline, making it easy for people who use assistive technologies (AT) and those who completely rely on keyboard navigation. The following keyboard shortcuts are supported by the Blazor 3D Chart component.
+
+Interaction Keys |Description
+-----|-----
+<kbd>Alt + J</kbd> | Moves the focus to the chart element.
+<kbd>Tab</kbd> |Moves the focus to the next element in the chart.
+<kbd>Shift + Tab</kbd> |Moves the focus to the previous element in the chart.
+<kbd>DownArrow</kbd> |Moves the focus to the data point right side from the selected point.
+<kbd>UpArrow</kbd> |Moves the focus to the data point right side from the selected point.
+<kbd>Left Arrow</kbd> |Moves the focus to the next series in our Chart component.
+<kbd>Right Arrow</kbd> |Moves the focus to the previous series in our Chart component.
+<kbd>Enter/Space</kbd> |Selects the data point in the series
+<kbd>Down/Left Arrow</kbd> |Moves the focus to the legend left side from the selected legend.
+<kbd>Up/Right Arrow</kbd> | Moves the focus to the legend right side from the selected legend.
+<kbd>Enter/Space</kbd> |Toggles the visibility of the corresponding series.
+<kbd>Ctrl + +</kbd> |Zoom in the chart.
+<kbd>Ctrl + -</kbd> |Zoom out the chart.
+<kbd>Down/Up Arrow</kbd> |Pans the chart vertically.
+<kbd>Left/Right Arrow</kbd> |Pans the chart horizontally.
+<kbd>R</kbd> |Reset the zoomed chart.
+<kbd>Ctrl + P</kbd> |Prints the Chart.
+
+## Ensuring accessibility
+
+The Blazor 3D Chart component's accessibility levels are ensured through an [axe-core](https://www.nuget.org/packages/Deque.AxeCore.Playwright) with playwright tests.
+
+The accessibility compliance of the Blazor 3D Chart component is shown in the following sample. Open the [sample](https://blazor.syncfusion.com/accessibility/chart) in a new window to evaluate the accessibility of the Blazor 3D Chart component with accessibility tools.
```

Comment:
@IlamathiSF3875 
Can we add demo for 3d chart and add link here. do not use chart.